index.bs

```diff
@@ -115,8 +115,100 @@ The <dfn for="a" element-attr>attributionreportto</dfn> attribute optionally dec
 The <dfn for="a" element-attr>attributionexpiry</dfn> attribute optionally defines the amount
 of time in milliseconds the attribution source should be considered for reporting.
 
-Issue: Need monkey patches passing attribution source in navigation, and a mechanism
-for validating the resulting document matches the attributiondestination.
+<h3 id="monkeypatch-navigation">Navigation</h3>
+
+The user agent must verify that an [=attribution source=] associated with a navigation
+results in a top-level navigation whose final url is [=same site=]
+[=attribution source/attribution destination=].
+
+<h4 id="monkeypatch-navigation-params">Navigation Params</h4>
+
+A <a spec="HTML">navigation params</a> struct has an item:
+
+: <dfn lt="navigation params attribution source" id="navigation-params-attribution-source">attribution source</dfn>
+:: an optional [=attribution source=] declared when initiating a navigation
+
+<h4 id="monkeypatch-navigate-algorithm">Navigate algorithm</h4>
+
+Modify the <a spec="html">navigate</a> algorithm to accept a new parameter
```

Comment:
```suggestion
Modify the <a spec="html">navigate</a> algorithm to accept a new optional parameter
```

? (and then "defaulting to null" if that's what you do for the navigation params item.)